src/libraries/System.Private.CoreLib/src/System/Numerics/Vector_1.cs

```diff
@@ -0,0 +1,1507 @@
+        /// <returns>True if the source vector was successfully copied to <paramref name="destination" />. False if
+        /// <paramref name="destination" /> is not large enough to hold the source vector.</returns>
+        public readonly bool TryCopyTo(Span<T> destination)
+        {
+            ThrowHelper.ThrowForUnsupportedVectorBaseType<T>();
+
+            if ((uint)destination.Length < (uint)Count)
+            {
+                return false;
+            }
+
+            Unsafe.WriteUnaligned<Vector<T>>(ref Unsafe.As<T, byte>(ref MemoryMarshal.GetReference(destination)), this);
+            return true;
+        }
+
+        /// <summary>Adds two vectors together.</summary>
+        /// <param name="left">The first source vector.</param>
+        /// <param name="right">The second source vector.</param>
+        /// <returns>The summed vector.</returns>
+        [Intrinsic]
+        public static unsafe Vector<T> operator +(Vector<T> left, Vector<T> right)
+        {
+            Vector<T> result = default;
+
+            for (nint index = 0; index < Count; index++)
+            {
+                result.SetElement(index, ScalarAdd(left.GetElement(index), right.GetElement(index)));
+            }
+
+            return result;
+        }
+
+        /// <summary>Subtracts the second vector from the first.</summary>
+        /// <param name="left">The first source vector.</param>
+        /// <param name="right">The second source vector.</param>
+        /// <returns>The difference vector.</returns>
+        [Intrinsic]
+        public static unsafe Vector<T> operator -(Vector<T> left, Vector<T> right)
+        {
+            Vector<T> result = default;
+
+            for (nint index = 0; index < Count; index++)
+            {
+                result.SetElement(index, ScalarSubtract(left.GetElement(index), right.GetElement(index)));
+            }
+
+            return result;
+        }
+
+        /// <summary>Multiplies two vectors together.</summary>
+        /// <param name="left">The first source vector.</param>
+        /// <param name="right">The second source vector.</param>
+        /// <returns>The product vector.</returns>
+        [Intrinsic]
+        public static unsafe Vector<T> operator *(Vector<T> left, Vector<T> right)
+        {
+            Vector<T> result = default;
+
+            for (nint index = 0; index < Count; index++)
+            {
+                result.SetElement(index, ScalarMultiply(left.GetElement(index), right.GetElement(index)));
+            }
+
+            return result;
+        }
+
+        /// <summary>Multiplies a vector by the given scalar.</summary>
+        /// <param name="value">The source vector.</param>
+        /// <param name="factor">The scalar value.</param>
+        /// <returns>The scaled vector.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static Vector<T> operator *(Vector<T> value, T factor)
```

Comment:
This one and the `T * Vector<T>` variant just below were doing `value * new Vector<T>(factor)`. I've changed that to be a `for` loop instead for the software fallback case.

Reply:
I am not sure whether I would bother with one-offs perf tweaks for the non-accelerated case like this at all. It won't move the needle. I would keep the non-accelerated case as simple as possible everywhere.